config_proc_macro/src/attrs.rs

```diff
@@ -0,0 +1,41 @@
+pub fn find_doc_hint(attrs: &[syn::Attribute]) -> Option<String> {
```

Comment:
Would be nice to have comments for the public methods!

Reply:
Added some comments inhttps://github.com/rust-lang/rustfmt/pull/3484/commits/efa45d2b902945a0c579428eb7d0ecfc8e37e826.

Reply:
@topecongiro thanks a lot for the comprehensive comments!